custom_components/tuya_local/devices/ledkia_fan_light.yaml

```diff
@@ -1,4 +1,7 @@
 name: Ledkia fan and light controller
+products:
+  - id: u5ovvhurbvynvspv
+    name: Light+Fan-new product
```

Comment:
Should I swap the top-level name with this one?

Reply:
I don't think it needs the "new product" bit, but "Light and fan" seems good for the top level name.

Reply:
Done!